__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require('supertest');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const db = require('../db/connection');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/categories', () => {
+    it('returns a categories array', () => {
+        return request(app).get('/api/categories')
+        .expect(200)
+        .then((res) => {
+            expect(Array.isArray(res.body.categories)).toBe(true);
+            expect(res.body.categories.length).toBe(4);
+        });
+    });
+    it('returns an array of categories with slug and description properties', () => {
+        return request(app).get('/api/categories')
+        .expect(200)
+        .then((res) => {
+            expect(res.body.categories[0].hasOwnProperty("slug")).toBe(true);
+            expect(res.body.categories[0].hasOwnProperty("description")).toBe(true);
+        });
+    });
+});
+
+describe('GET /api/reviews/:review_id', () => {
+    it('returns a review object', () => {
+        return request(app).get('/api/reviews/:1')
```

Comment:
you don't need to put the `:` in front of the id you're requesting in the path, so it should just be `/api/reviews/1`

Reply:
this applies to all of the tests on this endpoint!